.github/workflows/ci.yml

```diff
@@ -1,42 +1,56 @@
 name: CI
 
 on:
-  # Run CI on all pushes.
   push:
-    branches:
-      - master
-
   pull_request:
-    branches:
-      - master
+
+  # Run CI daily even without code changes to ensure tests pass with updated dependencies.
+  schedule:
+    - cron: '0 0 * * 5'
+  # See:
+  # * https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#scheduled-events
+  # * https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07
```

Comment:
I let it pass through in edalize, but since we're at it again: I don't think this documentation is particularly useful. We can keep it if you want, but then put it under the "# Run CI" comment above.

Reply:
I removed them.